corehq/ex-submodules/pillowtop/processors/form.py

```diff
@@ -73,7 +79,7 @@ def process_change(self, change):
                 )
 
 
-@quickcache(['domain', 'build_id'], timeout=60 * 60)
+@quickcache(['domain', 'build_id'], timeout=ONE_DAY, memoize_timeout=ONE_DAY)
```

Comment:
I also upped the caching of this and made it local. This is just marking an app as having a submission against it. It should never be cleared, so keeping it in local memory reduces queries to redis